libs/expo/betterangels/src/lib/screens/AddEditClient/RelevantContacts/Contact/index.tsx

```diff
@@ -59,7 +59,7 @@ export default function Contact(props: IContactProps) {
       null
```

Comment:
Hi @Davit-BetterAngels 

It worked great on iOS. 
As for android, the loading takes forever and I can't check with this device because of it... I will let you know when it's done.

Reply:
I was finally able to test the Android, and it worked good to me. Btw, I didn't experience what Mike experienced above (video)